content/en/docs/architecture/retester.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Retester"
+description: An overview of the retester implementation.
+---
+## What Is Retester?
+Retester is a tool that evaluate which pull requests should be retested, and then retest. 
+
+## How Do We Set up Retester?
+No configuration needed. Retester completely reuses [`tide`](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md) configuration, whatever onboards to Tide,
+ gets onboarded to retester too.
+Retester is aware of Prow and the concept of optional and required jobs. It only triggers retests on PRs
+ where at least one required Prow job is failing.
+
+### General configuration
```

Comment:
We have not implemented those yet.
Remove this section for now and add it back after supporting it.